src/aggregator/client/writer.go

```diff
@@ -288,7 +331,7 @@ func (w *writer) encodeUntimedWithLock(
 	default:
 	}
 
-	return errUnrecognizedMetricType
+	return ErrUnrecognizedMetricType
 }
 
 func (w *writer) encodeForwardedWithLock(
```

Comment:
Sanity check: the encoders will always get flushed eventually, right? Unless there's a `flush` call that hangs indefinitely, I assume? Is that case possible, or will timeouts save us?

Reply:
Thanks for explanation!